README.md

```diff
@@ -33,6 +33,11 @@ npm install -g @asyncapi/generator
     >**NOTE:** If you have to modify the [`dummy.yml`](https://github.com/asyncapi/generator/blob/master/test/docs/dummy.yml) file to develop your features, open a PR with the changes in the [asyncapi/generator](https://github.com/asyncapi/generator) repository.
 
 1. Generate output with watcher enabled: `npm run develop`.
+   
+   >**NOTE:** If your changes are not visible, this is maybe because the `ag` use the already installed `html-template` so you should use the `--install` option
+   > * run `ag https://raw.githubusercontent.com/asyncapi/generator/master/test/docs/dummy.yml ./ -o test/output --install --force-write --watch-template`
```

Comment:
better would be to add a new script for this, like `develop:install`

Reply:
what about 
```   
"//develop:install": "Force ag to install your local html-template source code",
"develop:install": " npm run develop -- --install",
```
@deberg ?

Reply:
@M3lkior perfect

Reply:
i pushed this ; let me know if it is ok for you